src/diffusers/models/autoencoder_kl.py

```diff
@@ -158,6 +189,108 @@ def decode(self, z: torch.FloatTensor, return_dict: bool = True) -> Union[Decode
 
         return DecoderOutput(sample=decoded)
 
+    def blend_v(self, a, b, blend_width):
+        for y in range(blend_width):
+            b[:, :, y, :] = a[:, :, -blend_width + y, :] * (1 - y / blend_width) + b[:, :, y, :] * (y / blend_width)
+        return b
+
+    def blend_h(self, a, b, blend_width):
+        for x in range(blend_width):
+            b[:, :, :, x] = a[:, :, :, -blend_width + x] * (1 - x / blend_width) + b[:, :, :, x] * (x / blend_width)
+        return b
+
+    def tiled_encode(self, x: torch.FloatTensor, return_dict: bool = True) -> AutoencoderKLOutput:
```

Comment:
Just a question, what use case does tiled encoding fulfills? Also, do we need blending during the encoding phase? If we used this as an autoencoder of a large image, I would have thought that blending during decoding would be enough to avoid seams between the tiles.

Reply:
Yeah, I meant that we don't need encoding for inference, and I don't think we'll train with very large images. It was just out of curiosity.

Reply:
I did need it for something. IIRC img2img with large images.

Reply:
Oh, you did img2img on huge images; cool, understood.